generate_xml_from_google_services_json.py

```diff
@@ -108,6 +108,33 @@ def construct_plist_dictionary(xml_root):
   return plist_dict
 
 
+def update_dict_keys(key_map, input_dict):
+  """Creates a dict from input_dict with the same values but new keys.
+
+  Two dictionaries are passed to this function: the key_map that represents a
+  mapping of source keys to destination keys, and the input_dict that is the
+  dictionary that is to duplicated, replacing any key that matches a source key
+  with a destination key. Source keys that are not present in the input_dict
+  will not have their destination key represented in the result.
+
+  In other words, if key_map is `{'old': 'new', 'foo': 'bar'}`, and input_dict
+  is `{'old': 10}`, the result will be `{'new': 10}`.
+
+  Args:
+    key_map: A dictionary of source keys to destination keys.
+    input_dict: The dictionary that is to be duplicated, replacing source keys
+      with the corresponding destination keys from key_map.
+
+  Returns:
+    A new map with updated keys.
```

Comment:
Same: Return type as in google style doc strings.

Reply:
Done